pkgs/by-name/op/openmm/package.nix

```
@@ -20,13 +20,13 @@
 
 stdenv.mkDerivation rec {
```

Meanwhile, you can use a lambda here (not applicable for Python yet)
```suggestion
stdenv.mkDerivation (finalAttrs: {
```